packages/babel-plugin-transform-es2015-block-scoping/src/index.js

```diff
@@ -90,9 +90,10 @@ function convertBlockScopedToVar(path, parent, scope, moveBindingsToParent = fal
     const parentScope = scope.getFunctionParent();
     const ids = path.getBindingIdentifiers();
     for (let name in ids) {
-      scope.removeOwnBinding(name);
+      let binding = scope.getOwnBinding(name);
+      if (binding) binding.kind = "var";
+      scope.moveBindingTo(name, parentScope);
```

Comment:
Oh I see, although I think it reads better if you didn't know what it did in moveBindingTo?

Although maybe it can be consistent with updateScopeInfo and do `if (!binding) continue;` instead